src/main/java/net/mcreator/ui/modgui/ModElementChangedListener.java

```diff
@@ -0,0 +1,139 @@
+/*
+ * MCreator (https://mcreator.net/)
+ * Copyright (C) 2012-2020, Pylo
+ * Copyright (C) 2020-2021, Pylo, opensource contributors
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ */
+
+package net.mcreator.ui.modgui;
+
+import javafx.embed.swing.JFXPanel;
+import net.mcreator.ui.component.JItemListField;
+import net.mcreator.ui.minecraft.JEntriesList;
+import net.mcreator.ui.minecraft.MCItemHolder;
+import net.mcreator.ui.modgui.codeviewer.ModElementCodeViewer;
+
+import javax.swing.*;
+import javax.swing.event.ChangeEvent;
+import javax.swing.event.ChangeListener;
+import javax.swing.event.DocumentEvent;
+import javax.swing.event.DocumentListener;
+import javax.swing.text.JTextComponent;
+import java.awt.*;
+import java.awt.event.*;
+
+/**
+ * Gets triggered whenever a change to a mod element is detected inside {@link ModElementGUI}
+ * @see ModElementGUI#elementUpdateListener
+ * @see ModElementCodeViewer#codeChangedListener

```

Comment:
This is no0t a big deal, but as most of the documentation uses `<p>blablabla</p>`, you should use this template too.

Reply:
???

Reply:
https://github.com/MCreator/MCreator/blob/86944d448cf8e4b03051679258997752c3ef9c0a/src/main/java/net/mcreator/plugin/PluginInfo.java#L95

Reply:
Not sure why using statement tag in docs is needed as it doesn't change the way the text is rendered, but I could add it.

Reply:
>it doesn't change the way the text is rendered

It just adds some space above & below the doc.

Reply:
SomeoneElse once said me to add them. 🤷‍♂️